stubs/app/Orchid/Screens/User/UserEditScreen.php

```diff
@@ -72,22 +82,25 @@ public function query(User $user): array
      */
     public function commandBar(): array
     {
-        return [
+        $btnLoginAs =
             Button::make(__('Impersonate user'))
                 ->icon('login')
                 ->confirm('You can revert to your original state by logging out.')
                 ->method('loginAs')
-                ->canSee($this->user->exists && \request()->user()->id !== $this->user->id),
+                ->canSee($this->editing && \request()->user()->id !== $this->user->id);
 
+        $btnRemove =
             Button::make(__('Remove'))
                 ->icon('trash')
                 ->confirm(__('Once the account is deleted, all of its resources and data will be permanently deleted. Before deleting your account, please download any data or information that you wish to retain.'))
-                ->method('remove'),
+                ->method('remove');
 
+        $btnSave =
             Button::make(__('Save'))
                 ->icon('check')
-                ->method('save'),
-        ];
+                ->method('save');
+
+        return $this->editing ? [$btnLoginAs, $btnRemove, $btnSave] : [$btnSave];
```

Comment:
`->canSee(...)` above in buttons definition hide buttons when not editing, so no need for this ternary operator.

Reply:
good catch.

```php
    return [
            Button::make(__('Impersonate user'))
                ->icon('login')
                ->confirm('You can revert to your original state by logging out.')
                ->method('loginAs')
                ->canSee($this->editing && \request()->user()->id !== $this->user->id),

            Button::make(__('Remove'))
                ->icon('trash')
                ->confirm(__('Once the account is deleted, all of its resources and data will be permanently deleted. Before deleting your account, please download any data or information that you wish to retain.'))
                ->method('remove')
                ->canSee($this->editing),

            Button::make(__('Save'))
                ->icon('check')
                ->method('save'),
        ];
```

Reply:
Remove button is only needed when editing.